### PR TITLE
Render "Property Bags" in result details.

### PR DESCRIPTION
**Pull request by jeffersonking**
Alternative to PR #401 (Issue #400). Instead of rendering plain JSON, this approach renders the Property Bag like this:

Screen Shot 2021-11-01 at 5 14 11 PM

### CODE REVIEW

**Comment by eddynaka** [comment]
@yongyan, we should do something similar for VS

File: src/panel/details.tsx [hunk 1]
```diff
@@ -100,7 +100,29 @@ interface DetailsProps { result: Result, height: IObservableValue<number> }
...
+                                    {Object.entries(rest).map(([key, value]) => {
+                                        return <Fragment key={key}>
+                                            <span className="ellipsis">{key}</span>
+                                            <span>{(() => {
+                                                if (value === null)
+                                                    return '—';
+                                                if (Array.isArray(value))
+                                                    return <span style={{ whiteSpace: 'pre' }}>{value.join('\n')}</span>;
+                                                if (typeof value === 'boolean')
+                                                    return JSON.stringify(value, null, 2);
```
Comment (eddynaka) [inline src/panel/details.tsx:117]: do we need to use `null, 2`? In this case, isn't this similar to JSON.stringify(value)? The same comment applies for the next stringify.

Comment (eddynaka) [reply src/panel/details.tsx:117]: or...wait...looking at the docs, it will add two spaces at the end. If that's correct, is that really needed?

Comment (jeffersonking) [reply src/panel/details.tsx:117]: The intent was to have pretty-printed JSON and the `null, 2` was for that.

**Review by eddynaka — APPROVED** [review]

File: src/panel/details.tsx (hunk 1 above)
Comment (jeffersonking) [reply src/panel/details.tsx:117]: "2" means "indent with 2 spaces". It is not adding spaces at the end.

**Review by sherryyshi — APPROVED** [review]

**Pull request merged**